README.md

```diff
@@ -1,32 +1,242 @@
-# Maven Dependencies
+# Adding Dependencies
```

Comment:
Will you be adding a lesson on Gradle dependencies too? If so, then maybe a better title for this might be Adding Maven Dependencies so it can be differentiated a little more clearly when displayed in Canvas. If not, then ignore this comment.

Reply:
updated to Adding Maven Dependencies.